### PR TITLE
Gradle conversion (plus license update)

### PR DESCRIPTION
**Pull request by DavidASeibert**
This makes the jar buildable using gradle, rather than maven. This is as close as I could get to using the common build files (v1.5.1), due to class cast and similar issues. This also includes a license update and a ZImbra version update to the latest community version (8.7).

### CODE REVIEW

File: src/main/resources/ecs.properties [hunk 1]
```diff
@@ -1 +1,23 @@
-��������� ������� ��� �������\��������� ������� � ��������
\ No newline at end of file
+## ECS Access Key
+ecs.access_key=131123009294999359@ecstestdrive.emc.com
+## ECS Secret Key
+ecs.secret_key=Ej816BbmYcy6SsBANKbSw1knSS8406aSKOsiV7+F
```
Comment (jasoncwik) [inline src/main/resources/ecs.properties]: Don't put your secret key in the template!

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,102 @@
+/*
```
Comment (jasoncwik) [inline build.gradle:1]: Probably don't need the whole license agreement in the header... Although that begs the question about what _should_ go in the source files.  Maybe ask Joe D'Angelo.

Comment (DavidASeibert) [reply build.gradle:1]: Will do. Right now, that's the only place the license agreement is shown, it's not in any other shipped code or in any binaries. I should probably fix that, make it match the object-client and other shipped binaries. What about the text - leave it EMC, or change to DELL EMC?

File: src/main/resources/ecs.properties (hunk 1 above)
Comment (DavidASeibert) [reply src/main/resources/ecs.properties]: Good catch, I will fix that.

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'ecsstoremanager'
```
Comment (twincitiesguy) [inline settings.gradle]: This should match the github project name

File: build.gradle (hunk 2 above)
Comment (DavidASeibert) [reply build.gradle:1]: Done, current copyright text was approved by Joe Dangelo.

File: settings.gradle (hunk 3 above)
Comment (DavidASeibert) [reply settings.gradle]: done.

**Pull request merged**